src/ansible_creator/resources/common/role/roles/run/README.md.j2

```diff
@@ -0,0 +1,57 @@
+{{ namespace|capitalize }}.{{ collection_name|capitalize }} Run Role
```

Comment:
In order to populate the namespace and collection name here, we'll need to read in the contents of the galaxy.yml file found in the root of the collection.